src/lib/shared-messages.js

```diff
@@ -20,5 +20,10 @@ export default defineMessages({
         defaultMessage: 'pop',
         description: 'Name of the pop sound, the default sound added to a sprite',
         id: 'gui.sharedMessages.pop'
+    },
+    defaultProjectTitle: {
+        id: 'gui.gui.defaultProjectTitle',
```

Comment:
Note: I left the id as `gui.gui.defaultProjectTitle` because it's already been translated. If we change the id to `gui.sharedMessages.defaultProjectTitle`, it won't be translated until we update translations.,

Reply:
👍